stock_picking_delivery_info_computation/tests/test_stock_picking_info_computation.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Tecnativa - Victor M.M. Torres
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.addons.sale.tests.test_sale_common import TestSale
+
+
+class TestStockPickingInfoComputation(TestSale):
+    def setUp(self):
+        super(TestStockPickingInfoComputation, self).setUp()
+        self.product_uom_unit = self.env.ref('product.product_uom_unit')
+        self.product_category_5 = self.env.ref('product.product_category_5')
+        self.product_pricelist_0 = self.env.ref('product.list0')
+        self.product_a = self.env['product.product'].create({
+            'name': 'Test product A',
+            'categ_id': self.product_category_5.id,
+            'standard_price': 100.0,
+            'list_price': 200.0,
+            'type': 'consu',
+            'uom_id': self.product_uom_unit.id,
```

Comment:
No need of specifying all of these things. Only relevant fields like name, type, weight or volume